src/shared/utilities/vsCodeUtils.ts

```diff
@@ -109,7 +106,7 @@ export async function activateExtension(
         }
     }
 
-    const extension = vscode.extensions.getExtension(extId)
+    const extension = vscode.extensions.getExtension<T>(extId)
```

Comment:
i think this is the only change needed. Then the caller can decide whether it needs the whole extension or just wants the `.exports`.

Doesn't look like anything depended on this so doesn't matter too much I guess.